FreshRSS/FreshRSS
# Added: Display option for "My labels"

Closes #5882



How to test the feature manually:

1. toggle the checkbox in Display configs
2. see the "my labels" in article footer in normal/global view

Pull request checklist:
